foundation/foundation-mda/src/test/resources/specifications/data-records-generation.feature

```diff
@@ -214,21 +214,26 @@ Feature: Generating pipeline resources
     Then the "PySparkPipeline" pipeline POM has a dependency on "example-data-records-python"
 
   @pipeline-generation @code-generation
-  Scenario: Pipeline pyproject includes data module by default
+  Scenario: Pipeline generates file store environment variables necessary for accessing a single configured file store
     Given a project named "example"
-    And a "data-flow" pipeline using "data-delivery-pyspark"
-    And a dictionary and 0 or more record models
-    When the profile "data-delivery-pyspark-pipeline" is generated
-    Then the pyproject.toml file has a dependency on "example-data-records"
+    And a "data-flow" pipeline using "data-delivery-spark" named "SparkPipeline" and a file store named "s3TestModelOne"
+    When the profile "data-delivery-spark-pipeline" is generated
+    Then the "s3TestModelOne_FS_PROVIDER", "s3TestModelOne_FS_ACCESS_KEY_ID", and "s3TestModelOne_FS_SECRET_ACCESS_KEY" configurations are generated to access the "s3TestModelOne" file store
 
-  @pipeline-generation @module-generation
-  Scenario: Pipeline pyproject includes data module by default - two language support
+  @pipeline-generation @code-generation
+  Scenario: Pipeline does not generate file store environment variables for accessing a file store when it does not have a file store configured
     Given a project named "example"
-    And a "data-flow" pipeline using "data-delivery-spark"
-    And a "data-flow" pipeline using "data-delivery-pyspark"
-    And a dictionary and 0 or more record models
-    When the profile "data-delivery-pyspark-pipeline" is generated
-    Then the pyproject.toml file has a dependency on "example-data-records-python"
+    And a "data-flow" pipeline using "data-delivery-spark" named "SparkPipeline"
+    When the profile "data-delivery-spark-pipeline" is generated
+    Then the "s3TestModelOne_FS_PROVIDER", "s3TestModelOne_FS_ACCESS_KEY_ID", and "s3TestModelOne_FS_SECRET_ACCESS_KEY" configurations are not generated
+
+  @pipeline-generation @code-generation
+  Scenario: Pipeline generates file store environment variables necessary for accessing multiple configured file stores
+    Given a project named "example"
+    And a "data-flow" pipeline using "data-delivery-spark" named "SparkPipeline" and two file stores named "s3TestModelOne" and "s3TestModelTwo"
```

Comment:
A: I think this might read a little cleaner if we just let _"...and two file stores..."_ actually be a second `And` clause.  Might even allow us to combine the single file store step and double file store step with something like "And the file store(s) "filestore1", "filestore2"

Reply:
Refactored.